### PR TITLE
Throw exception when private key generation fails. (bunq/sdk_php#102)

### PR DESCRIPTION
**Pull request by OGKevin**
[//]: # (Thanks for opening this pull request! Before you proceed please make sure that you have an issue that explains what this pull request will do.
         Make sure that all your commits link to this issue e.g. "My commit. \(bunq/sdk_php#<issue nr>\)".
         If this pull request is changing files that are located in "src/Model/Generated" then this pull request will be closed as these files must/can only be changed on bunq's side.)
         
## This PR closes/fixes the following issues:
[//]: # (If for some reason your pull request does not require a test case you can just mark this box as checked and explain why it does not require a test case.)
 - Closes bunq/sdk_php#102
    - [x] Tested

### CODE REVIEW

**Comment by OGKevin** [comment]
@sandervdo please 👀

File: src/Security/KeyPair.php [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 /**
  * Represents an OpenSSL key pair.
  *
  */
 class KeyPair
 {
+    /**
+     * Error constants
+     */
+    const ERROR_PRIVATE_KEY_GENERATION_FAILED = <<<'EOT'
```
Comment (sandervdo) [inline src/Security/KeyPair.php]: Why the quotes around `EOT`?

File: src/Security/KeyPair.php [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+Generating a new private key failed. See "http˜://php.net/manual/en/function.openssl-pkey-new.php" for more info.
```
Comment (sandervdo) [inline src/Security/KeyPair.php]: `http~`, new protocol? 😛

Comment (OGKevin) [reply src/Security/KeyPair.php]: :see_no_evil:

**Comment by OGKevin** [comment]
Thx @sandervdo :hugs:

**Comment by OGKevin** [comment]
Ack, thx @sandervdo

File: src/Security/KeyPair.php [hunk 3]
```diff
@@ -1,12 +1,20 @@
 <?php
 namespace bunq\Security;
+use bunq\Exception\BunqException;
```
Comment (andrederoos) [inline src/Security/KeyPair.php]: enter after namespace please

**Pull request merged**